src/services/YotiCallbackProcessor.ts

```diff
@@ -189,6 +197,61 @@ export class YotiCallbackProcessor {
   			throw new AppError(HttpCodesEnum.SERVER_ERROR, "Failed to send to IPV Core", { shouldThrow: true });
   		}
 
+  		// Document type objects to pass into TxMA event F2F_CRI_VC_ISSUED
+
+  		let documentInfo;
+  		let docName: any;
+
+  		if (documentFields.document_type === "PASSPORT") {
+  			docName = "passport";
```

Comment:
Do you think we could put these in a an enum/include them in one that already exists?

Reply:
I added a DocumentTypes enum for the 4 doc types, is that what you meant?